assets/js/googlesitekit/data/utils.test.js

```diff
@@ -787,4 +793,81 @@ describe( 'data utils', () => {
 			} );
 		} );
 	} );
+
+	describe( 'createValidatedAction', () => {
+		it( 'should throw an error if a validator function is not provided', () => {
+			const actionCreator = noop;
+
+			return expect( () => createValidatedAction( null, actionCreator ) )
+				.toThrow( 'a validator function is required.' );
+		} );
+
+		it( 'should throw an error if an action creator function is not provided', () => {
+			const validator = noop;
+
+			return expect( () => createValidatedAction( validator ) )
+				.toThrow( 'an action creator function is required.' );
+		} );
+
+		it( 'should throw an error if action creator function is a generator object', () => {
+			const validator = noop;
+			function* actionCreator() { }
+
+			return expect( () => createValidatedAction(
+				validator,
+				// this has to be called to return generator object from generator function
+				actionCreator()
+			) )
+				.toThrow( 'an action creator function is required.' );
+		} );
```

Comment:
This test doesn't really make sense because it tests the logic that has already been tested in the previous test where we called the `createValidatedAction` function without the second parameter (in other words passed `undefined`). I suggest removing this test because it is redundant.

Reply:
Apologies @eugene-manuilov ! I managed to confuse myself here with some previous changes :facepalm: 

I've fixed this test now to correctly check if the validator is a generator (which is the whole point of the `invariant` call in the previous file!)